datashuttle/datashuttle_class.py

```diff
@@ -868,10 +868,36 @@ def setup_ssh_connection(self) -> None:
 
         if verified:
             ssh.setup_ssh_key(self.cfg, log=True)
+            # 449
+            confirmed_path = (
+                input(
+                    f"Confirm the full central path (default: {self.cfg['central_path']}): "
+                )
+                or self.cfg["central_path"]
+            )
+            self.cfg["central_path"] = confirmed_path
+            if not self._check_write_permissions(confirmed_path):
```

Comment:
This could be moved under `_setup_rclone_central_ssh_config` so that the connection is set up fully, but then the user is informed if the central path they have chosen does not have the permission error (it is possible to set up the connection even if the central path is wrong). So the order can be like

1) setup the connection fully and message to indicate the connection is okay.
2) raise if the central path does not have permissions